code-of-conduct.html

```diff
@@ -11,6 +11,9 @@
 	</HEAD>
 	<BODY>
 		<nav>
+					<div class="homeLogo">
+						<a href="https://www.realdevsquad.com/"><img src="img/logos/favicon.ico"  width="35px" height="auto"></a>
+					</div>
```

Comment:
Not properly indented. Can we please remove those extra spaces?

Reply:
Extra spaces removed.